demo/lit-html/demo-lit-html.js

```diff
@@ -16,12 +27,6 @@ export default class DemoLitHtml extends BaseElement {
   static template(html, { ifDefined }) {
     return ({ emoji, message }) => {
       return html`
-        <style>
```

Comment:
Simply changing this inline `<style>` tag usage so we can have a stricter content security policy on this particular _demo_. To be clear, this is not production code.